data/pigui/libs/buttons.lua

```diff
@@ -0,0 +1,141 @@
+-- Copyright © 2008-2020 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+local Engine = require 'Engine'
+local ui = require 'pigui.baseui'
+local pigui = Engine.pigui
+
+--
+-- Function: ui.imageButton
+--
+-- ui.imageButton(icon, size, frame_padding, bg_color, fg_color, tint_color, tooltip)
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--
+--   icon          - image to place on button, e.g. from ui.theme.icons
+--   size          - size of button, Vector2
+--   frame_padding - number
+--   bg_color      - Color, for background
+--   fg_color      - Color, for forground
+--   tint_color    - Color, 
+--   tooltip       - string, mouseover text, will be used as ID, must be unique, append "##uniqueID" if needed
+--
+-- Returns:
+--
+--   boolean - true if button was clicked
+--
+function ui.imageButton(icon, size, frame_padding, bg_color, tint_color, tooltip)
+	local uv0, uv1 = get_icon_tex_coords(icon)
+	ui.withID(tooltip, function()
+		local res = pigui.ImageButton(ui.icons_texture, size, uv0, uv1, frame_padding, bg_color, tint_color)
+	end)
+	return res
+end
```

Comment:
As is this file!

Reply:
Same!
![image](https://user-images.githubusercontent.com/7342077/94460240-6cf25b80-0186-11eb-92f5-7a0e62a2a75e.png)